### ISSUE TITLE
Ancestral state reconstruction function

### ISSUE THREAD

**Issue by ScottClaessens** [ScottClaessens/coevolve#86]
Not a priority, but it would be nice to have a function that plots ancestral state reconstructions for a given trait. We could do this on the latent scale (as in our NHB paper) and/or on the observation scale.

**Comment by ErikRingen** [comment]
Yep, I was also thinking this and had some scrap-paper code version of this for a while when I was working on the manuscript. I found it was actually very straightforward because of the way we arrange the tree segments: the eta indices should line up exactly with the nodes of the tree.

**Comment by ScottClaessens** [comment]
Yes, exactly. We can also use the code from the authority paper for this. I think this isn't something we should try to get done for the initial release though.

**Comment by SimonGreenhill** [comment]
yes please!

**Comment by HedvigS** [comment]
yes please :D

**Comment by ErikRingen** [comment]
open PR for those interested in this feature, would be great to get feedback on whether the capabilities from the vignette are what people actually want: https://github.com/ScottClaessens/coevolve/pull/110
